gems/pending/appliance_console.rb

```diff
@@ -461,6 +462,32 @@ module ApplianceConsole
         dbhost, database, region = ApplianceConsole::Utilities.db_host_database_region
         press_any_key
 
+      when I18n.t("advanced_settings.db_replication")
+        say("#{selection}\n\n")
+
+        options = {
+          "Initialise Database Replication Master"  => "master",
```

Comment:
Also I'm not sure about the wording. The database is already "initialized".

Maybe "Configure Server as Primary" and "Configure Server as Standby"?